tests/test_distro.py

```diff
@@ -687,7 +691,6 @@ def test_empty_release(self):
     def test_unknowndistro_release(self):
         self._setup_for_distro(os.path.join(TESTDISTROS, 'distro',
                                             'unknowndistro'))
-
```

Comment:
With my picky hat on: No objections but my vote for moving to a commit dedicated to whitespace changes.

Reply:
Reverted.